src/telemetry/opentelemetry.ts

```diff
@@ -132,6 +114,26 @@ const instrumentations = [
 api.diag.setLogger(new api.DiagConsoleLogger(), api.DiagLogLevel.INFO);
 
 export const tracer = (serviceName: string) => {
+  const traceExporter = isProd
```

Comment:
Do we need to set sample rate for the spans?

Reply:
I've set it now in the env variables